client/jetpack-connect/authorize.js

```diff
@@ -242,9 +242,7 @@ export class JetpackAuthorize extends Component {
 		const { alreadyAuthorized, authApproved, from } = this.props.authQuery;
 		return (
 			this.isSso() ||
-			( 'woocommerce-services-auto-authorize' === from ||
-				( ! config.isEnabled( 'jetpack/connect/woocommerce' ) &&
-					'woocommerce-setup-wizard' === from ) ) ||
+			( 'woocommerce-services-auto-authorize' === from || 'woocommerce-setup-wizard' === from ) || // Auto authorize the old WooCommerce setup wizard only.
```

Comment:
Nitpicky, but we could use the includes pattern used elsewhere in this component to make it a tiny bit faster to grok.

```suggestion
			( includes( [ 'woocommerce-services-auto-authorize', 'woocommerce-setup-wizard' ], from ) || // Auto authorize the old WooCommerce setup wizard only.
```